### PR TITLE
Added file field to thesis defense

### LINKED ISSUE #3906: Add attachment file field to Thesis Defense
# Story

As an editor, I can add a PDF to a Thesis Defense content item.

### PR DESCRIPTION
**Pull request by dnkemper**
# How to test

1. blt ds --site=genetics.grad.uiowa.edu (since grad.uiowa.edu is having db issues rn)
2. make sure can add file field to thesis defense content type
3. boom

### CODE REVIEW

**Comment by dnkemper** [comment]
@pyrello does it matter how the pdf file is displayed?

**Comment by pyrello** [comment]
Just a link is fine.

**Comment by dnkemper** [comment]
@pyrello thank you!

File: config/grad.uiowa.edu/user.role.webmaster.yml [hunk 1]
```diff
@@ -286,7 +286,7 @@ permissions:
   - 'override scholar authored by option'
   - 'override scholar authored on option'
   - 'override scholar published option'
-  - 'override scholarship authored by option'
+  - 'override scholarship authored on option'
```
Comment (richardbporter) [inline config/grad.uiowa.edu/user.role.webmaster.yml]: oops

Comment (richardbporter) [reply config/grad.uiowa.edu/user.role.webmaster.yml]: This should be undone.

**Review by richardbporter — APPROVED** [review]
Ok I think grad.uiowa.edu is syncing correctly now. Hope I didn't miss a permission. 🤞🏻 

Tested file. 👍🏻

**Pull request merged**